run_evolution.sh

```
@@ -0,0 +1,39 @@
+# This script runs the command in the background to avoid shell timeouts
+
+# Check if at least one argument is provided
+if [ $# -eq 0 ]; then
+    echo "Usage: $0 \"your prompt\" [--evolve] [--generations N] [--population N] [other options...]"
+    echo "Example: $0 \"Create a game concept\" --evolve --generations 3 --population 10"
+    exit 1
+fi
+
+# Ensure we're in the right directory
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd "$SCRIPT_DIR"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)